sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1648,6 +1648,12 @@ public void onClick(DialogInterface dialog, int which) {
      */
     private synchronized void goToUnknownAppsSettings(ReleaseDetails releaseDetails) {
         Intent intent;
+        if (mForegroundActivity == null) {
+
+            /* We should not open settings screen if app is in background now. */
+            AppCenterLog.warn(LOG_TAG, "App is in background mode, we could not open setting screen");
```

Comment:
I'd avoid "we". it doesn't make logs clear (imho).
```suggestion
            AppCenterLog.warn(LOG_TAG, "The application is in background mode, the settings screen could not be opened.");
```